### PR TITLE
feat(HMS-776): Add provider to provisioning wizard

### PR DESCRIPTION
**Pull request by amirfefer**
needed for https://github.com/RHEnVision/provisioning-frontend/pull/195

### CODE REVIEW

File: src/Components/ImagesTable/ImageLink.js [hunk 1]
```diff
@@ -74,7 +75,9 @@ const ImageLink = ({ imageId, isExpired, isInClonesTable }) => {
 
   if (
     hasProvisioning &&
-    (image.imageType === 'aws' || image.imageType === 'ami')
+    (image.imageType === 'gcp' ||
```
Comment (ezr-ondrej) [inline src/Components/ImagesTable/ImageLink.js:78]: @amirfefer I don't think this we want to expose it to everyone yet, this is not done feature, it needs quite some work to be completed. could we do it only if a feature flag is enabled?

Comment (ezr-ondrej) [reply src/Components/ImagesTable/ImageLink.js:78]: I'd vote for removing the gcp for now, just adding the providerType so we are sure Provisioning Wizard does not crash.

Comment (amirfefer) [reply src/Components/ImagesTable/ImageLink.js]: agree, i removed the gcp for now, adding only `provider`
in a follow-up PR , i'll add a feature flag for gcp provider.

**Review by ezr-ondrej — DISMISSED** [review]
LGTM :+1:

@croissanne @lucasgarfield could you take a look pls? :)

**Review by ezr-ondrej — APPROVED** [review]

**Review by croissanne — APPROVED** [review]

**Pull request merged**